src/value.jl

```diff
@@ -217,6 +217,14 @@ Base.:<(::Missing, ::CatValue) = missing
 
 # JSON of CatValue is JSON of the value it refers to
 JSON.lower(x::CatValue) = JSON.lower(get(x))
```

Comment:
BTW, I wonder whether it would be possible to get rid of that JSON dependency thanks to DataAPI. See https://github.com/JuliaData/CategoricalArrays.jl/pull/96 for a discussion. Or maybe JSON can just use one of the printing methods from Base, not sure.